charts/app-stable-diffusion/templates/deployment.yaml

```diff
@@ -36,62 +37,60 @@ spec:
           volumeMounts:
             {{- if .Values.HFCacheVolume.enabled }}
             - name: huggingface-cache
-              mountPath: /root/.cache/huggingface
+              mountPath: {{ $huggingfaceDir }}
             {{- end }}
           env:
-          {{- range $k, $v := .Values.deployment.env }}
+          {{- range $k, $v := .Values.api.env }}
             - name: {{ $k }}
               value: {{ $v | quote }}
           {{- end }}
-          command: [ "huggingface-cli" ]
+          command: [ "/bin/sh", "-c" ]
           args:
-            - download
-            - {{ .Values.model.modelHFName }}
-            {{- if .Values.model.modelFiles }}
-            - {{ .Values.model.modelFiles }}
-            {{- end }}
+{{/*            huggingface Docker was not updated for a long time so it could not recognize the dir param */}}
+            - "pip install -U 'huggingface_hub[cli]' && huggingface-cli download {{ .Values.model.modelHFName }} {{ if .Values.model.modelFiles }} {{ .Values.model.modelFiles }} {{ end }} --local-dir /home/worker/app/huggingface-cache/"
```

Comment:
nit: we could create own repo & build image for this case

Reply:
I also thought about it, but! the effort can be smaller, to make it stable we can just fix the version of cli library, 
WDYT?

Reply:
sure, we could do that